tests/python/unittest/test_random.py

```diff
@@ -871,14 +871,15 @@ def test_randint_generator():
             buckets = np.array(buckets, dtype=dtype).tolist()
             probs = [(buckets[i][1] - buckets[i][0]) / float(scale) for i in range(5)]
             generator_mx = lambda x: mx.nd.random.randint(low, high, shape=x, ctx=ctx, dtype=dtype).asnumpy()
-            verify_generator(generator=generator_mx, buckets=buckets, probs=probs)
+            verify_generator(generator=generator_mx, buckets=buckets, probs=probs, nrepeat=100, alpha=0.01)
+            # Scipy uses alpha = 0.01 for testing discrete distribution generator
```

Comment:
can you also point to documentation or code link.

Reply:
Done